streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -241,12 +241,14 @@ public static void cleanStateAfterTest(final EmbeddedKafkaCluster cluster, final
      * @param <K>                 Key type of the data records
      * @param <V>                 Value type of the data records
      */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
Curious: why should we need this in the integration test utils? There's no need to make something public for later use, since we can just make it public if/when we need it to be.

Reply:
It's for discoverability -- I often use Intellij auto-complete to see what methods are available -- if not public, it's not listed...